### PR TITLE
Request to merge contact-section to main

### PR DESCRIPTION
**Pull request by serengia**
I request to merge the **contact-section** branch to the **main** branch.
---
The contact-section branch contains the section html markup with the corresponding css.

### CODE REVIEW

**Comment by serengia** [comment]
I however do not agree with the highlighted Stylelint. Please let me know what you think.

**Comment by mckent05** [comment]
> I however do not agree with the highlighted Stylelint. Please let me know what you think.

- It is important the checks pass before I can review, I'd suggest you take a close and keen look at the issues and try to fix them 🙏 . Thank you. Until you do, I'll have to invalidate your project ❌ ❌

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Serengia and @z4cope , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

You have submitted a project, that has some linter checks failing, please look into fixing these issues and feel free to submit again when they pass. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by serengia** [comment]
Hello @mckent05 . Thanks for the feedback. Please advise on what to do because I have been fixing the errors on my local environment only for them to change once I push to GitHub.

File: index.html [hunk 1]
```diff
@@ -345,14 +341,46 @@ <h3>Skills</h3>
...
   </main>
+  <footer>
+    <div class="footer-container row">
+      <div class="form-container">
+        <h2 class="heading-secondary heading-secondary--blue">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/myyvwvbw" method="POST">
+          <input type="text" name="name" id="name" placeholder="Full name" required />
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly add `maxlength="30"` attribute to the name field so it can only accept up to 30 characters. 😄

File: index.html [hunk 2]
```diff
@@ -345,14 +341,46 @@ <h3>Skills</h3>
...
+          <input type="email" name="email" id="email" placeholder="Email address" required />
+          <textarea name="message" id="user-message" cols="30" rows="10" placeholder="Enter Text Here"
```
Comment (jasonfelice) [inline index.html]: - [ ] Please add `maxlength="500"` attribute to the message field.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Serengia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Great Design✔️
The form is working✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ndorrh , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**